src/model_signing/_cli.py

```diff
@@ -57,6 +57,14 @@
     help="Location of the signature file to verify.",
 )
 
+# Decorator for the commonly used option to ignore certain paths
+_ignore_paths_option = click.option(
+    "--ignore-paths",
+    metavar="IGNORE_PATHS",
+    multiple=True,
+    default=[],
```

Comment:
Let's instead declare the type and remove the default (since it's implicit as an empty list, given it's not required). See ~_private_key_option` below.

Reply:
So this should be the type of a single argument (`pathlib.Path`), even though it gets repeated.

Click takes repeated arguments and places them in a tuple, but in the rest of the code we use them as a list. To be fully correct, we should change to `Sequence[pathlib.Path]` in the function arguments, but that can be a different PR. Right now, there seems to be an implicit conversion from `tuple` to `list` (most likely because the code does not use any API that exists in only one of these types).

Reply:
Done.